src/rating-calculator/components/ChartRecordHeadRow.tsx

```diff
@@ -12,18 +12,21 @@ const MessagesByLang = {
     song: 'Song',
     nextGoal: 'Next Goal',
     nextRating: '+Rating',
+    newRating: 'New Rating',
   },
   [Language.zh_TW]: {
     num: '#',
     song: '歌曲',
     nextGoal: '下個\n目標',
     nextRating: 'R 值\n加分',
+    newRating: 'New Rating',
```

Comment:
Will need help on the CN and KR text here....

Reply:
I think we can just use `CommonMessages[lang].rating`. I prefer making the column header shorter so it takes less width

Reply:
Happy to implement the above - currently the width is 65px since 70px is for `+Rating`. I'll refactor to use `CommonMessages[lang].rating` instead

Reply:
Updated as requested